apps/right-to-rent-check/behaviours/caseworker-email.js

```diff
@@ -5,8 +5,12 @@ const path = require('path');
 
 const getSubjectWithTotalTenants = (model, translate) => {
   const totalTenants = model.tenants.length;
-  return translate('email.caseworker.subject.prefix') +
-    totalTenants + translate('email.caseworker.subject.suffix');
+  let subject = translate('email.caseworker.subject.prefix') +
+    totalTenants;
```

Comment:
Why are these on two lines, IMO it's easier to read on one line.

Reply:
you're right it should be on one line, I'll amend